src/Apps/Order/Components/TransactionDetailsSummaryItem.tsx

```diff
@@ -66,6 +70,15 @@ export class TransactionDetailsSummaryItem extends React.Component<
     }
   }
 
+  buyerTotalDisplayAmount = order => {
```

Comment:
```suggestion
  buyerTotalDisplayAmount = () => {
    const { order } = this.props
```

Reply:
should also do the same kind of thing for the other methods here. There's no reason to be using `any` all over the place by accident.